src/Symfony/Component/EventDispatcher/Attribute/AsEventListener.php

```diff
@@ -24,6 +24,10 @@ public function __construct(
         public ?string $method = null,
         public int $priority = 0,
         public ?string $dispatcher = null,
+        /** @var string|array{0: string, 1: string}|null */
```

Comment:
IIRC, we use `@param` instead to describe types for constructor promoted properties needing them.